generator/plugins/proto2gql/plugin.go

```diff
@@ -63,20 +86,26 @@ func (p *Plugin) normalizeGenerateConfigPaths() error {
 		p.config.Files[i].ProtoPath = normalizedPath
 
 	}
+
 	return nil
 }
+
 func (p *Plugin) prepareFileConfig(fileCfg *ProtoFileConfig) error {
```

Comment:
do you really need the error on the returning?

Reply:
removed.